packages/syntax-core/src/LinkButton/LinkButton.module.css

```diff
@@ -1,5 +1,6 @@
 .linkButton {
   text-decoration: none;
+  box-sizing: border-box;
```

Comment:
@somethiiing small nit - ideally we alphabetize these properties when possible.

Reply:
got it, fixed